integrations/extensions/starter-kits/google-custom-search/README.md

```diff
@@ -33,9 +35,20 @@ If you want to make a _new_ Assistant using this starter kit, take the following
 - Use (`./basic`) directory for basic search functionality and (`./advanced`) directory for extending the basic search with advanced filtered search.
 - Use the OpenAPI specification to [build a custom extension](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-build-custom-extension#building-the-custom-extension).
 - [Add the extension to your assistant](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-add-custom-extension) using the API key you obtained in the pre-requisites above.
-- Select "Custom Search" or "Custom Search Site Restricted" as the endpoint at this step. If search is over more than 10 different web sites (which can include an unlimited number of pages on each site), we would recommend using "Custom Search Site Restricted" since it doesn't have a limit on the number of queries you can run per day.
 - [Upload the Actions JSON file](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-admin-backup-restore#backup-restore-import).
-- Use either method listed in [Configuring Your Actions Skill to use an Extension](https://github.com/watson-developer-cloud/assistant-toolkit/blob/master/integrations/extensions/README.md#configuring-your-actions-skill-to-use-an-extension) to configure the actions you uploaded to invoke the custom extension you built. Set the `query` parameter to the `query_text` session variable and the `cx` parameter to an `cx` session variable you obtained during the pre-requisites step and `num` parameter to `num_of_results` session variable or default value to 3.
+- Under "variables"/"set by you" (within the Actions page), set the variable with name `cx` and set to the ID for the programmable search engine that you got in the Pre-Requisite Steps (replacing `0123456789abcdef0`, which is the dummy/placeholder value in the actions file).
+- Use either method listed in [Configuring Your Actions Skill to use an Extension](https://github.com/watson-developer-cloud/assistant-toolkit/blob/master/integrations/extensions/README.md#configuring-your-actions-skill-to-use-an-extension) to configure the actions you uploaded to invoke the custom extension you built.
+   - In the "Search" action that is included in both the basic and advanced versions, use the following configuration on the step that uses the extension:
+      - Click on "Edit extension" at the bottom of the step
+      - Select *Custom Search* or *Custom Search Site Restricted* as the endpoint at this step.  See the "Background" section of this document for information about the difference between the two.
+      - Set the `query` parameter to the `query_text` session variable.
+      - Set the `cx` parameter to the `cx` session variable.
+      - Set the `num` parameter to the `num_of_results` session variable (which is set to 3 in the Actions JSON file).
```

Comment:
Maybe good to clarify that `num` is an optional parameter? Since the user has to click the dropdown of the optional parameters to set it 
<img width="858" alt="Screen Shot 2022-10-11 at 10 19 36 AM" src="https://user-images.githubusercontent.com/6188953/195157938-66c5e69f-dc1f-4fc2-a6d0-42a6d9ac15ea.png">

Reply:
Good idea!  Will do

Reply:
Done!

Reply:
Is it in a new commit? For some reason I don't see it

Reply:
Looks like I did not get that commit in -- thank you for catching that.  I have tried again and it is in there now.